cmd/kwild/config/test_data/config.toml

```diff
@@ -51,8 +49,20 @@ without_gas_costs = true
 # Toggle to disable nonces for transactions and queries
 without_nonces = false
 
-# KWILD Sqlite database file path
```

Comment:
remove `without_gas_costs` and `without_nonces` from config.toml as they are genesis params

Reply:
I think I didn't remove this before merge.  We can sneak that into a docs PR.